bls/src/main/cpp/threshold-v1.cpp

```diff
@@ -0,0 +1,35 @@
+#include "threshold-v1.hpp"
+using namespace std;
+namespace bls {
+
+    PrivateKey Threshold::PrivateKeyRecover(const std::vector<PrivateKey>& sks, const std::vector<vector<uint8_t>>& ids) {
+        std::vector<bls::Bytes> idsVec;
+        idsVec.reserve(ids.size());
+        for (size_t i = 0; i < ids.size(); i++) {
+            idsVec.emplace_back(ids[i]);
+        }
+        return Threshold::PrivateKeyRecover(sks, idsVec);
+    }
```

Comment:
The `Bytes` class does not work well with SWIG tool because this class has `const` members and `vector<Bytes>` creates many compile errors because are no default constructor or assignment operators.  

So in `Java` we have Uint8VectorVector that will map to `std::vector<vector<uint8_t>>` and in functions like this be converted to `vector<Bytes>`.